backend/entityservice/models/run.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, project_id, threshold, name, notes):
     @staticmethod
     def from_json(data, project_id):
         # Get optional fields from JSON data
```

Comment:
Should modify the comment: the `threshold` is not optional, as specified in the swagger file.